src/index.js

```diff
@@ -184,7 +184,7 @@ class FlipperReduxInspectorPlugin extends FlipperPlugin {
               actions={(
                 <>
                   <Button onClick={this.handleClear}>🗑️ Clear</Button>
-                  <Button onClick={this.handleHelp}>⛑️ Intergrade with your app</Button>
+                  <Button onClick={this.handleHelp}>⛑️ Integration guide</Button>
```

Comment:
Maybe also capitalize the `g` in `Guide`?

Reply:
Sure! Done